README.md

```diff
@@ -1,13 +1,13 @@
-# Ethereum Protocol Fellowship Study Group
+# Ethereum Protocol Fellowship Study Group (EPFsg)
 
-This repository is dedicated to coordination and knowledge base for EPFsg. It hosts the Protocol Wiki, a community effort to build a technical knowledge base about the Ethereum core protocol. 
+This repository is dedicated to fostering coordination and developing a knowledge base for the EPFsg. It hosts the Protocol Wiki, a community effort to build a technical knowledge base focused on the Ethereum core protocol.

```

Comment:
```suggestion
This repository is dedicated to fostering coordination and developing a knowledge base for the EPFsg. It hosts the Protocol Wiki, a community effort to build a technical knowledge base about the Ethereum core protocol.

```
I think "focused on"  implies the wiki has special interest in Ethereum protocol (possibly among other things); about is more direct that Ethereum is entire subject of the wiki.